applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -439,21 +439,23 @@ CONFIG_BRIDGE_MIGRATE_PRE_2_7_0
 .. _CONFIG_BRIDGE_MIGRATE_VERSION_1:
```

Comment:
Alignment for this option and the option above needs to be fixed. 
![image](https://github.com/user-attachments/assets/df365aeb-b512-41ee-82fc-9c832adb2b79)

Reply:
@umapraseeda please check if fixed now